### ISSUE TITLE
Create a Markdown File for Procurement Assistantship

### ISSUE THREAD

**Issue by SuzyQuant** [TaoFruit/instar#81]
Create a Markdown File how external people can help in writing procurement.

**Comment by bjornmage** [comment]
I believe Harvard Catalyst's Grant Central has something like this or other Ivy League schools have Research areas where they pull from grants.gov and encourage scientists to come on and do grant writing and pick projects they want to do.  we definitely should have an area like this.  think about the right shape of this and what it should look like based on other leading NGOs that offer the same.

Also, as said below: think about the branding around ours.  We already are going to have "Dream Maker" which is owned by Dream Limited and provides a dashboard for people to write grants, and manages the whole process. instead of duplicating that here, it makes sense to steer people towards the Dream Maker program, I think?

**Comment by bjornmage** [comment]
Research follow-up on the procurement/grant-writing assistance area:

Harvard Catalyst's "Grant Central" is mostly *human expertise + education* (review services, bootcamps, checklists, outcomes data) — not a software feed. Peers (Pivot-RP for academia, Instrumentl for nonprofits, Foundation Directory) license a database and differentiate on the human layer.

Recommendation: the INSTAR site hosts a lightweight **discovery + free resource shelf** (grant-writing guidance, Specific Aims tips, funder-type guides) for SEO/credibility, and routes all actual writing/management to the **Dream Maker** program rather than duplicating it. This is now the **Research → Funding** page in PR #217. A standalone procurement-assistance markdown can fold into that resource shelf.

**Comment by bjornmage** [comment]
PROPOSAL slice is ready for your input — MR git.developerdojo.org/INSTARLab/Website/-/merge_requests/61 (targets demo, pipeline green). Deliverable is plan/procurement-assistantship-proposal.md — proposal only, no site changes.

It lays out three options grounded in what exists today (/research/funding/ as the designated home, /research/opportunities/ verified-only + empty-state pattern, /fellowship/ external-handoff pattern, and the no-live-forms rule):
- A: resource shelf folded into the Funding page, handoff to Dream Maker (recommended now — matches your steer in this thread)
- B: standalone procurement-assistantship page (only if you want a dedicated URL)
- C: Dream Maker referral track (right end state, needs Dream Limited coordination)

Decision ask for you (Q1–Q7 in the doc): confirm whether a live program exists vs aspirational shelf; roles/eligibility and who reviews contributions; the Dream Maker boundary + link destination; whether specific solicitations should ever be listed; branding (keep procurement assistantship?) and placement (A vs B); timeline. Every assumption is marked as needing your confirmation — nothing was invented. Nothing closed; implementation waits on your adjudication.
